washboard-ui/CONTRIBUTING.md

```diff
@@ -25,7 +25,16 @@ Enable `corepack` and install `yarn` using the following commands:
 corepack enable
 yarn install
 ```
+### Building ts files
 
+Run the following command to build typescript files in @wasmcloud/.. workspace package
+to avoid the following error: 
+
+`Unable to resolve path to module '@wasmcloud/lattice-client-react'  import/no-unresolved`
+
+```bash
+yarn run turbo:lint
```

Comment:
```suggestion
yarn run turbo:build
```

Running the build step will be better here since it is required by `turbo:lint` in `turbo.json` which is what solves the underlying issue. I'm surprised that the following `yarn run dev` command doesn't prevent this error.

Reply:
Got it :+1: ,additional context of the issue was to get rid of `Name` field since a unique `ID` already differentiates the component, just want to make sure I can go ahead with doing so.